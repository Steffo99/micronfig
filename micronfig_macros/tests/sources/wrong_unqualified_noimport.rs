micronfig::config! {
	GARASAUTO: String > PathBuf,
}

fn main() {
	println!("{:?}", GARASAUTO())
}
