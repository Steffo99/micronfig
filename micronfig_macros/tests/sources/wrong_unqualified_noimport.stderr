error[E0412]: cannot find type `PathBuf` in this scope
 --> tests/sources/wrong_unqualified_noimport.rs:2:22
  |
2 |     GARASAUTO: String > PathBuf,
  |                         ^^^^^^^ not found in this scope
  |
  = help: consider importing this struct:
          std::path::PathBuf

error[E0412]: cannot find type `PathBuf` in this scope
 --> tests/sources/wrong_unqualified_noimport.rs:2:22
  |
2 |     GARASAUTO: String > PathBuf,
  |                         ^^^^^^^ not found in this scope
  |
help: consider importing this struct
  |
1 + use std::path::PathBuf;
  |
